scripts/release-processor-images.mjs

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env -S node
+
+
+// This script releases indexer processor images to docker hub https://github.com/aptos-labs/aptos-indexer-processors.
+// It does so by copying the images from aptos GCP artifact registry to docker hub.
+
+// Usually it's run in CI, but you can also run it locally in emergency situations, assuming you have the right credentials.
+// Before you run this locally, check one more time whether you can trigger a CI build instead which is usually easier and safer.
+// You can do so via the Github UI or CLI:
+// E.g: gh workflow run copy-processor-images-to-dockerhub.yaml
+//
+// If that doesn't work for you, you can run this script locally:
+//
+// Prerequisites when running locally:
+// 1. Tools:
+//  - docker
+//  - gcloud
+//  - node (node.js)
+//  - crane - https://github.com/google/go-containerregistry/tree/main/cmd/crane#installation
+//  - pnpm - https://pnpm.io/installation
+// 2. docker login - with authorization to push to the `aptoslabs` org
+// 3. gcloud auth configure-docker us-west1-docker.pkg.dev
+// 4. gcloud auth login --update-adc
+//
+// Once you have all prerequisites fulfilled, you can run this script via:
+// GIT_SHA=${{ github.sha }} GCP_DOCKER_ARTIFACT_PROCESSOR_REPO_US="${{ secrets.GCP_DOCKER_ARTIFACT_REPO }}" ./docker/release-processor-images.mjs --language=rust --wait-for-image-seconds=1800
```

Comment:
This usage example uses both env vars and args, but as far as I can tell the script just uses args. Can we update this?

Reply:
Would you mind elaborating? it uses GIT_SHA and GCP_DOCKER_ARTIFACT_PROCESSOR_REPO_US in the script

Reply:
So this invocation:
```
GIT_SHA=${{ github.sha }} GCP_DOCKER_ARTIFACT_PROCESSOR_REPO_US="${{ secrets.GCP_DOCKER_ARTIFACT_REPO }}" ./docker/release-processor-images.mjs --language=rust --wait-for-image-seconds=1800
```

It uses both env vars (e.g. `GIT_SHA`) and args (e.g. `--language`). This is sort of confusing, can we just use one of env vars or args?

Reply:
because `GCP_DOCKER_ARTIFACT_PROCESSOR_REPO_US` is github secret, so I feel it make sense to make it `env`. In most cases this will triggered via github workflow:
```
- name: Release Images
        env:
          FORCE_COLOR: 3 # Force color output as per https://github.com/google/zx#using-github-actions
          GIT_SHA: ${{ inputs.GIT_SHA }}
          GCP_DOCKER_ARTIFACT_PROCESSOR_REPO_US: ${{ secrets.GCP_DOCKER_ARTIFACT_REPO_US }}
        run: ./scripts/release-processor-images.mjs --language=${{ inputs.processor_language }} --version-tag=${{ inputs.version_tag }} --wait-for-image-seconds=3600
```

Reply:
Okay sounds good